src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java

```diff
@@ -101,11 +100,10 @@ public void loggedInUserOnDesktopCanVoteForFirstReply() {
     firstReplyUpvoteButtonClickAddsAnUpvoteAndSecondClickRemovesTheUpvote();
   }
 
-  @Test(groups = "discussions-loggedInUserOnDesktopCanUpvote", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-loggedInUserOnDesktopCanUpvote")
   @Execute(asUser = User.USER_3)
-  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
```

Comment:
if you remove browser = Browser.FIREFOX, test will be run on mobile skin by Jenkins. See [here](https://wikia-inc.atlassian.net/wiki/pages/viewpage.action?pageId=71303284) why it happens like that